mkdocs.yml

```diff
@@ -68,6 +68,9 @@ nav:
   - "Topics": 'topics'
 
 markdown_extensions:
+  - pymdownx.inlinehilite
+  - pymdownx.snippets:
+      dedent_subsections: True
```

Comment:
Extension configuration here

Reply:
Oh, maybe this is what I'm missing! I'll have to give this a shot.